srcs/Comms.cpp

```diff
@@ -243,6 +243,26 @@ int	noticeCommand(Server &serv, Message &attempt)
 	return (0);
 }
 
+int	inviteCommand(Server &serv, Message &attempt)
+{
+	Channel* chn = serv.get_channelPtr(attempt.params[1]);
+	Client* cln = serv.get_clientPtr(attempt.params[0]);
+	if (cln == NULL)
+	{
+		return (-1); //Unknown server
+ 	}
+	
+
+	if (chn == NULL)
+	{
+		return (-2); //Unknown server
+ 	}
+	// send invite for channel chn to client cln
```

Comment:
There also should be checked if the client who invites is on the channel, if not, sending ERR_NOTONCHANNELto the client who invited. 
Also, should check if channel is invite only, and if it is, then only operator of the channel can invite people, if channel is invite only and client who invites isnt a operator of that channel, then sending ERR_CHANOPRIVSNEEDED. 
And have to check if invited user is already on the channel, if it is then sending ERR_USERONCHANNEL.

Or the send invite function will take care about this conditions and replies?